operations/helm/charts/mimir-distributed/templates/query-scheduler/query-scheduler-servmon.yaml

```
@@ -0,0 +1,59 @@
+{{- if .Values.query_scheduler.enabled }}
+{{- with .Values.serviceMonitor }}
+{{- if .enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "mimir.resourceName" (dict "ctx" $ "component" "query-scheduler") }}
+  {{- with .namespace }}
+  namespace: {{ . }}
```

Can you make this default to the release namespace?

```suggestion
  namespace: {{ . | default $.Release.Namespace }}
```